app/cdap/components/NamespaceAdmin/Tabs.tsx

```diff
@@ -96,13 +97,23 @@ const Tabs: React.FC = () => {
         >
           Drivers
         </NavLink>
+        <span className={classes.separator}>|</span>
+        <NavLink
+          exact
+          to={`${baseNSPath}/scm`}
```

Comment:
is scm an okay url?

Reply:
i guess its fine but this might be better long form to make it less confusing in the future?

Reply:
will change in future if requested